registry.yaml

```diff
@@ -795,6 +795,22 @@ packages:
   format_overrides:
   - goos: darwin
     format: zip
+- type: github_release
+  repo_owner: Peltoche
+  repo_name: lsd
+  asset: 'lsd-{{.Version}}-{{.Arch}}-{{.OS}}.{{.Format}}'
+  description: The next gen ls command
+  format: tar.gz
+  format_overrides:
+  - goos: windows
+    format: zip
+  replacements:
+    amd64: x86_64
+    darwin: apple-darwin
+    linux: unknown-linux-musl # Set musl as default. If architecture matchs, this can be executable without relying on external libraries.
```

Comment:
From my end(maxOS), worked correctly.

```
$ cat aqua-test.yaml                                                        ?[chaspy/lsd]
registries:
- name: standard
  type: local
  path: registry.yaml

packages:
- name: Peltoche/lsd
  registry: standard
  version: 0.20.1 # renovate: depName=Peltoche/lsd

$ aqua -c aqua-test.yaml i --test
```

And the CI passed on Linux.

But I don't have windows machine and it seems to tweak something. (maybe replacement for windows is needed)
https://github.com/Peltoche/lsd/releases/tag/0.20.1
<img width="396" alt="Screen Shot 2021-09-18 at 7 52 11" src="https://user-images.githubusercontent.com/10370988/133861579-4ac7b691-4c2a-4ff8-9820-03eb1ab6be02.png">

@suzuki-shunsuke 
What do you think? I'm not sure msvc compiler 😅

Reply:
Currently, aqua doesn't support Windows.
https://github.com/suzuki-shunsuke/aqua#note-windows-isnt-supported
I don't have Windows too.
We don't have to consider Windows for now.